velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -418,11 +417,31 @@ class NonNumericMinMaxAggregateBase : public exec::Aggregate {
         return;
       }
       if (!accumulator->hasValue() ||
-          compareTest(accumulator->compare(decoded, i))) {
+          compareTest(compare(accumulator, decoded, i))) {
         accumulator->write(baseVector, indices[i], allocator_);
       }
     });
   }
+
+ private:
+  FOLLY_ALWAYS_INLINE static int32_t compare(
+      SingleValueAccumulator* accumulator,
+      DecodedVector& decoded,
+      vector_size_t index) {
+    VELOX_CHECK_NOT_NULL(accumulator);
```

Comment:
This check is redundant and per-row. Let's remove.

Reply:
Fixed